bazel/setup_clang.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
@htuch you might want to use this script for developer tools setup as well.

Reply:
Yep, I see the value of this, ta.